.npmignore

```diff
@@ -0,0 +1 @@
+/mongosh
```

Comment:
Now that we have it as a dependency, I think we can remove this?

Reply:
Yes, thank you very much for noticing all the middle/test/debug code here! 🍬 I was experimenting so many times and I knew it will be a bit of a mess in this PR..)